mrjob/yarnemr.py

```diff
@@ -269,11 +284,17 @@ def _usable_clusters(self, valid_clusters, invalid_clusters,
         # list of (available_mb, cluster_id)
         cluster_list = []
 
+        # If we are limiting cluster counts then list all active clusters
+        if self._opts['max_pool_cluster_count']:
```

Comment:
is there a case where this key doesn't exist?

Reply:
It will default as `None`.